src/controllers/pagoPAController.ts

```diff
@@ -29,6 +30,7 @@ export default class PagoPAController {
     // tslint:disable-next-line:max-union-size
     | IResponseErrorValidation
     | IResponseErrorInternal
+    | IResponseErrorTooManyRequests
```

Comment:
where is this coming from? do you see anything below returning a 429 that needs to be forwarded to the app?

Reply:
Not directly. It come from profileService.getProfile.